Index: src/main/java/module-info.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>module com.univr.employeemanager {\r\n    requires javafx.controls;\r\n    requires javafx.fxml;\r\n    requires json;\r\n    requires com.google.gson;\r\n\r\n\r\n    opens com.univr.employeemanager to javafx.fxml, com.google.gson;\r\n    exports com.univr.employeemanager;\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/module-info.java b/src/main/java/module-info.java
--- a/src/main/java/module-info.java	(revision 368635c54dabc77d3e4fa9aeae48b45afacdf6be)
+++ b/src/main/java/module-info.java	(date 1671189887542)
@@ -1,7 +1,7 @@
 module com.univr.employeemanager {
     requires javafx.controls;
     requires javafx.fxml;
-    requires json;
+    //requires json;
     requires com.google.gson;
 
 
